lib/socket.js

```diff
@@ -84,7 +85,7 @@ class Transport extends EventEmitter {
 
 const socketFactory = (connect) => {
   const resultConnectFn = (...options) => {
-    const callback = common.extractCallback(options);
+    const callback = common.cbExtract(options);
```

Comment:
This callback comes from `jstp.net.connect` and `jstp.net.connectAndInspect` and tls/ws/wss analogs. Try to remove this callack in `test/node/transport-net-tcp.js` for example.